pol-core/pol/uoscrobj.cpp

```diff
@@ -4396,7 +4396,8 @@ BObjectImp* UWeapon::get_script_member_id( const int id ) const
     return new String( attribute().name );
     break;
   case MBR_HITSCRIPT:
-    return new String( hit_script_.relativename( tmpl->pkg ) );
+    // bad method name? nullptr makes it return the fullpath
+    return new String( hit_script_.relativename( nullptr ) );
```

Comment:
How about we make a new method called `fullname` or `absolutename` or `package_qualified_name` ?

Reply:
(it can call relativename(nullptr) )